content/general/concepts/xml/xml.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: 'XML'
+Description: 'XML (Extensible Markup Language) is used to define the content and structure of data in a document.'
+Subjects:
+  - 'Code Foundations'
+  - 'Data Science'
+Tags:
+  - 'Data Structures'
+  - 'Development'
+  - 'Tags'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-analyst'
+---
+
+XML (Extensible Markup Language) is used to define the content and structure of arbitrary data in a document. It is a human-readable text-based markup language used to store and transmit many different types of data, from individual records in a database, to things as complex as a Microsoft Word document. XML is primarily defined in the [XML 1.0 Specification](http://www.w3.org/TR/REC-xml) from the [World Wide Web Consortium](https://www.w3.org/standards/) (W3C). XML was designed for simplicity, general application, and usability across the Internet. While document-based, it is widely used to store all types of arbitrary data structures.
+
+Many computer languages, such as [Python](https://www.codecademy.com/resources/docs/python) or [Java](https://www.codecademy.com/resources/docs/java), provide their own libraries or modules for processing XML documents. XML has proved to be a popular format for [serializing](https://www.codecademy.com/resources/docs/general/serialization) objects.
+
+An XML document can consist of the following pieces:
+
+## Elements
+
+An element is defined with an opening tag with an arbitrary name enclosed with angle brackets `<...>` and a closing tag defined by the same name enclosed in angle brackets and a leading slash `</...>`. Between these two tags can be an arbitrary number of other elements, or a text value. An element can combine opening and closing tags by enclosing the tag name with angle brackets and a trailing slash `<.../>`. A well-formed XML document has only one outermost element that contains the rest of the document's content.
+
+The following are all valid elements:
+
+- `<tag>text</tag>`
+- `<notext />`
+- `<outer><inner /></outer>`
```

Comment:
Would these look better in a multiline code snippet? (e.g. \```pseudo)

Reply:
They probably do.